react-native.config.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  project: {
+    ios: {},
+    android: {},
+  },
+  assets: ['./assets/fonts'],
```

Comment:
assets/fonts should not be a top level folder. make fonts a subdirectory and place the font folders inside

Reply:
I am not sure how to do this. Is it like below?
```
fonts: ['./assets/fonts] , 
```
Else please help me with this on Saturday meeting.

Reply:
The path reference is correct, but I think you named the folder ‘assets/fonts’ instead of nesting it properly. You don’t need to make any changes here, you just need to create the fonts subdirectory inside assets and rename the top level. You may need to adjust the font references elsewhere afterwards.